deploy.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# This script is not currently meant to be executed from within the repository.
+# Copy or symlink it to some deployment directory.
+
+set -ex
+
+# Defaults for production.
+# Example for development:
+#   PORT=8100 OPENFISCA_PORT=2000 OPENFISCA_CONFIG=development PUBLIC_URL=http://next.mes-aides.sgmap.fr ./deploy.sh aah
+TARGET_BRANCH=${1:-master}  # demo
+PORT=${PORT:-8000}  # 8100
+OPENFISCA_PORT=${OPENFISCA_PORT:-12000}  # 2000
+OPENFISCA_CONFIG=${OPENFISCA_CONFIG:-mes-aides}  # development
+PUBLIC_URL=${PUBLIC_URL:-https://mes-aides.gouv.fr}  # http://next.mes-aides.sgmap.fr
+
+# Install Mes Aides
+if ! cd mes-aides-ui
+then
+    git clone https://github.com/sgmap/mes-aides-ui.git
+    cd mes-aides-ui
+fi
+
+git fetch origin $TARGET_BRANCH
+git checkout origin/$TARGET_BRANCH
+
+rm -rf node_modules # work around libsass bindings being never found
+npm install
+grunt build
+
+# Stop Mes Aides
+killall --user `whoami` node || echo 'No server was running'
```

Comment:
If we kill all node processes that belong to the user, I guess it means they can be only one instance per user.
So we create a new user each time we want to deploy a branch ?